ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/MovableBehaviour.java

```diff
@@ -0,0 +1,20 @@
+package com.github.migi_1.Context.model.entity;
+
+import com.jme3.math.Vector3f;
+
+/**
+ * class for handling the creation of a certain moving behaviour for an entity. 
+ * @author Damian
+ *
+ */
+public class MovableBehaviour {
```

Comment:
I would name this class MoveBehaviour, because now it sounds like it's a behaviour that can be moved, instead of a behaviour for moving.